CMakeLists.txt

```diff
@@ -156,6 +156,20 @@ ecbuild_add_option( FEATURE GPU_STATIC
                     DEFAULT ${GPU_STATIC_DEFAULT}
                     DESCRIPTION "Compile GPU library as static library")
 
+ecbuild_add_option( FEATURE ETRANS
+                    DEFAULT OFF
+                    DESCRIPTION "Include Limited-Area-Model Transforms" )
+
+ecbuild_add_option( FEATURE PROGRAMS
+                    DEFAULT ON
+                    DESCRIPTION "Build src/programs" )
```

Comment:
Why do we need an option to disable the programs?

If there is a good reason to do so, we should be consistent with other packages (e.g. eccodes) which use the [feature "BUILD_TOOLS" ](https://github.com/ecmwf/eccodes/blob/develop/CMakeLists.txt#L107-L109 )for this.

Reply:
See answer to comment by Olivier above